src/nvim/math.c

```diff
@@ -78,13 +78,15 @@ int xctz(uint64_t x)
 }
 
 /// Count number of set bits in bit field.
-int popcount(uint64_t x)
+unsigned xpopcount64(uint64_t x)
```

Comment:
I don't think we care to add the `64` to our helper function:
```suggestion
unsigned xpopcount(uint64_t x)
```